client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -36,6 +39,18 @@ export default function Header( { isEditorSidebarOpened, toggleGeneralSidebar }:
 				<span className="gutenboarding__header-site-heading">
 					{ siteTitle ? siteTitle : NO__( 'Create your site' ) }
 				</span>
+				<Button onClick={ () => setDomainPopoverVisibility( s => ! s ) }>
+					Pick a domain
+					{ isDomainPopoverVisible && (
+						<Popover
+							/* Prevent interaction in the domain picker from affecting the popover */
+							onClick={ e => e.stopPropagation() }
+							onKeyDown={ e => e.stopPropagation() }
```

Comment:
Kinda weird we have to do that manually and don't get it out-of-the-box from Popover :thinking:

Reply:
`Popover` _inside_ another interactive element like `Button` _may_ indicate a bad practice?

Reply:
Maybe :thinking: